source/uwp/Visualizer/ViewModel/DocumentViewModel.cs

```diff
@@ -238,6 +238,22 @@ public static void InitializeRenderer(AdaptiveHostConfig hostConfig)
             // Custom resource resolvers
             _renderer.ResourceResolvers.Set("symbol", new MySymbolResourceResolver());
 
+            _renderer.OverrideStyles = new ResourceDictionary();
+
+            Style textInputBorderStyle = new Style(typeof(TextBox));
```

Comment:
what is this change about again?

Reply:
That is a change in the visualizer to darken the border of input elements